features/cart/shopping_cart/seeing_detailed_information_about_cart.feature

```diff
@@ -1,15 +1,17 @@
 @shopping_cart
-Feature: Seeing order items in cart
-    In order to see details about my order items
+Feature: Seeing detailed information of cart
+    In order to see detailed information of my cart
     As a Visitor
-    I want to be able to see my cart order items
+    I want to be able to see those information of my cart
 
     Background:
         Given the store operates on a single channel in "United States"
         And the store has a product "T-shirt banana" priced at "$12.54"
 
     @api
-    Scenario: Viewing content of my cart
+    Scenario: Viewing items and items total of my cart
         When I add 5 of them to my cart
         And I check items in my cart
         Then my cart should have 5 items of product "T-shirt banana"
+        And I check details of my cart
+        Then my cart should have "$62.70" items total
```

Comment:
These 2 steps should be merged into one:
```suggestion
        And my cart should have "$62.70" items total
```

or create the second scenarios for checking the items total. I know I was suggesting something else, but I didn't know we need to use different endpoints for these checks